tests/test_packaging/test_unpackage_dataset.py

```diff
@@ -172,7 +175,7 @@ def test_unpackage_dataset_failed():
         unpackage_dataset(
             packaged_file_path, RSA_KEYS_DIRECTORY, OUTPUT_DIRECTORY
         )
-    except InvalidTarFileContents:
+    except UnpackagingError:
```

Comment:
You can test that you get the expected exception here with:
```py
with pytest.raises(UnpackagingError):
    unpackage_dataset(
            packaged_file_path, RSA_KEYS_DIRECTORY, OUTPUT_DIRECTORY
        )
```
Instead of the try/pass ^^